README.md

```diff
@@ -2,7 +2,7 @@
 
 This plugin uses ```snmpv3``` with ```MD5``` + ```AES``` to check a lot of different values on your Synology DiskStation.
 
-This check plugin needs ```pysnmp``` to be installed on your system. You can install it with: ```pip install pysnmp```
+This check plugin needs ```pysnmp``` to be installed on your system. You can install it with: ```pip install easysnmp```
```

Comment:
I run into an error while installing ```easysnmp```. The documentation gave me the hint that ```easysnmp``` depends on ```net-snmp``` (https://easysnmp.readthedocs.io/en/latest/)

```suggestion
This check plugin needs ```easysnmp``` to be installed on your system. You can install it with: ```pip install easysnmp``` (Note that ```easysnmp``` needs ```net-snmp``` you might need to install ```libsnmp-dev``` and ```snmp-mibs-downloader``` on your system)
```